business/definitionService.js

```diff
@@ -504,10 +504,10 @@ class DefinitionService {
     }
   }
 
-  async _ensureUrls(coordinates, definition) {
```

Comment:
push this level of function down to clearlydefined.js or the appropriate summarizer. Should not need to access the raw results after the `summarizeAll` call.